seqr/views/apis/report_api.py

```diff
@@ -342,7 +343,27 @@ def _process_saved_variants(saved_variants_by_family, family_individual_affected
             variant['inheritance_models'] = inheritance_models
             for gene_id in potential_compound_het_gene_ids:
                 potential_com_het_gene_variants[gene_id].append(variant)
+            for tag in variant['discovery_tag_names']:
+                for guid in variant.get('discovery_tag_guids_by_name', {}).get(tag):
+                    potential_mnvs[tag][guid].append(variant)
+        mnv_genes = set()
+        for variants_by_tag_guid in potential_mnvs.values():
+            mnvs = next((variants for variants in variants_by_tag_guid.values() if len(variants) > 2), None)
+            if not mnvs:
+                continue
+            parent_mnv = next((v for v in mnvs if not v.get('populations')), mnvs[0])
+            nested_mnvs = [v for v in mnvs if v.get('populations')]
```

Comment:
this will be cleaner if you instead explicitly check that it isn't the parent mnv (i.e. `v['variantId'] != parent_mnv['variantId']`)